docs/13-walkthroughs/2-deploying-doom.md

```diff
@@ -224,18 +224,7 @@ The dos-games package has now been removed from your cluster.
 
 #### Example
 
-```sh
```

Comment:
Didn't notice it before but:

On line 223 the same hanging chad thing from before, and
On line 21 there is an extra back-tick (`) at the end of the line and
Did we want to convert the rest of the terminal output blocks to ascii2html iframes?